moztelemetry/standards.py

```diff
@@ -83,3 +83,46 @@ def mau(dataframe, target_day, past_days=28, future_days=10, date_format="%Y%m%d
     filtered = filter_date_range(dataframe, act_col, min_activity, max_activity,
         sub_col, min_submission, max_submission)
     return count_distinct_clientids(filtered)
+
+def snap_to_beginning_of_week(date, weekday_start="Sunday"):
```

Comment:
The `date` param name is the same as an imported module - please change to something else (I suggest `day`).